src/api/eventTypes.js

```diff
@@ -13,13 +13,27 @@
 import type { Message, Stream, UserPresence } from './modelTypes';
 
 export class EventTypes {
-  static heartbeat: 'heartbeat' = 'heartbeat';
+  static alert_words: 'alert_words' = 'alert_words';
   static message: 'message' = 'message';
-  static presence: 'presence' = 'presence';
+  static delete_message: 'delete_message' = 'delete_message';
   static stream: 'stream' = 'stream';
+  static update_message: 'update_message' = 'update_message';
+  static subscription: 'subscription' = 'subscription';
+  static presence: 'presence' = 'presence';
+  static muted_topics: 'muted_topics' = 'muted_topics';
+  static realm_emoji: 'realm_emoji' = 'realm_emoji';
+  static realm_filters: 'realm_filters' = 'realm_filters';
   static submessage: 'submessage' = 'submessage';
-  static update_message_flags: 'update_message_flags' = 'update_message_flags';
+  static update_global_notifications: 'update_global_notifications' = 'update_global_notifications';
+  static update_display_settings: 'update_display_settings' = 'update_display_settings';
   static user_status: 'user_status' = 'user_status';
+  static realm_user: 'realm_user' = 'realm_user';
+  static realm_bot: 'realm_bot' = 'realm_bot';
+  static reaction: 'reaction' = 'reaction';
+  static heartbeat: 'heartbeat' = 'heartbeat';
+  static update_message_flags: 'update_message_flags' = 'update_message_flags';
+  static typing: 'typing' = 'typing';
+  static user_group: 'user_group' = 'user_group';
 }
```

Comment:
Originally the order of these had matched the order of the event type definitions below. (Which is the main place they're used.)

Then that got out of sync, though. These `type` values stayed alphabetical -- previously both had been, which I don't recall if I'd intended -- but the type definitions didn't.

Fine to add these others, but perhaps simplest to keep them in alphabetical order. The order of appearance in `eventToAction` is a bit chaotic, as it's governed to a large extent by things like how we group them for the `return { ...event, type: actionTypeOfEventType[event.type] }` case.